### PR TITLE
⭐️ switch to cnspec service

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-This cookbook installs the Mondoo Client on Linux servers.
+This cookbook installs the Mondoo `cnquery` and `cnspec` on Linux servers.
 
 It does:
 
 * Installs the signed `mondoo` package
-* Registers the client with Mondoo Platform
-* Enables the systemd service
+* Login `cnquery` and `cnspec` with Mondoo Platform
```
Comment (tas50) [inline README.md:11]: Login is confusing in this wording

File: recipes/deb.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Cookbook:: mondoo
 # Recipe:: deb
 #
-# Copyright:: 2021, Mondoo, Inc, All Rights Reserved.
+# Copyright:: 2022, Mondoo, Inc, All Rights Reserved.
```
Comment (tas50) [inline recipes/deb.rb]:
```suggestion
# Copyright:: 2021-2023, Mondoo, Inc, All Rights Reserved.
```

File: recipes/default.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # Cookbook:: mondoo
 # Recipe:: default
 #
-# Copyright:: 2021, Mondoo, Inc, All Rights Reserved.
+# Copyright:: 2022, Mondoo, Inc, All Rights Reserved.
```
Comment (tas50) [inline recipes/default.rb]:
```suggestion
# Copyright:: 2021-2023, Mondoo, Inc, All Rights Reserved.
```

File: recipes/rpm.rb [hunk 4]
```diff
@@ -2,7 +2,7 @@
 # Cookbook:: mondoo
 # Recipe:: rpm
 #
-# Copyright:: 2021, Mondoo, Inc, All Rights Reserved.
+# Copyright:: 2022, Mondoo, Inc, All Rights Reserved.
```
Comment (tas50) [inline recipes/rpm.rb]:
```suggestion
# Copyright:: 2021-2023, Mondoo, Inc, All Rights Reserved.
```

File: README.md (hunk 1 above)
Comment (chris-rock) [reply README.md:11]: Any other wording is recommended

Comment (tas50) [reply README.md:11]: It should just be register. Registers works. Logins does not.

Comment (chris-rock) [reply README.md:11]: The command has been renamed to login and register command does not exist anymore. I am not sure it makes sense to confuse users with an old term then.

Comment (tas50) [reply README.md:11]: It's confusing text, but it's not the end of the world.

**Pull request merged**